### PR TITLE
AND/OR skip evaluation for 1.4

### PR DESCRIPTION
**Pull request by ashtul**
cherrypick #1158 for 1.4

### CODE REVIEW

File: src/value.h [hunk 1]
```diff
@@ -260,6 +260,9 @@ static inline int RSValue_BoolTest(RSValue *v) {
     case RSValue_String:
       return v->strval.len != 0;
     case RSValue_RedisString: {
+      if (v->rstrval == NULL) {
```
Comment (MeirShpilraien) [inline src/value.h:263]: So we are not going to cherry pick the is_null?

Comment (ashtul) [reply src/value.h:263]: @MeirShpilraien will do. waiting to finalize on master

Comment (MeirShpilraien) [reply src/value.h:263]: So we do not really need this line then right?

Comment (ashtul) [reply src/value.h:263]: @MeirShpilraien 
it failed without it
Will add isnull here as well

Comment (ashtul) [reply src/value.h:263]: In 1.4, `evalProperty` always return `RSValue_RedisString`. If field does not exist, it contains `null`

**Comment by ashtul** [comment]
replaced by #1172

**Pull request closed without merging**